dotnet/src/Agents/Core/Chat/SelectionStrategy.cs

```diff
@@ -12,6 +12,19 @@ namespace Microsoft.SemanticKernel.Agents.Chat;
 /// </summary>
 public abstract class SelectionStrategy
 {
+    /// <summary>
+    /// Flag indicating if an agent has been selected (first time).
+    /// </summary>
+    protected bool HasSelected { get; private set; }
+
+    /// <summary>
+    /// An optional root agent for initial selection.
+    /// </summary>
+    /// <remarks>
+    /// Useful to avoid latency in initial agent selection.
+    /// </remarks>
+    public Agent? RootAgent { get; set; }
```

Comment:
Are we sure about `RootAgent` naming? It wasn't obvious for me that this property sets the agent which will be selected initially. Maybe something like `InitiatingAgent` or `InitialAgent` will describe the purpose of this property better.

Reply:
Updated.